tools/e2e-commons/bin/tunnel.js

```diff
@@ -34,6 +34,24 @@ yargs( hideBin( process.argv ) )
 	.help( 'h' )
 	.alias( 'h', 'help' ).argv;
 
+/**
+ * This allows overriding the tunnel with a custom tunnel like ngrok.
+ * Useful when running e2e tests locally and you want to use a tunnel that's
+ * closer to you than the localtunnel instance.
+ *
+ * For example:
+ * ```
+ * TUNNEL_URL=https://somethingsomething.ngrok.io npm run test-e2e:start
```

Comment:
I think this should say `test-e2e:run` instead of `test-e2e:start`. start is for starting the tunnel and we don't need that since we'll be providing our own URL.